src/render.rs

```diff
@@ -38,3 +40,32 @@ pub fn render(item: &str, opts: Opts, extra_opts: ExtraOpts) -> String {
 
     rendered_content
 }
+
+/// Escape a math block `item` into a delimited string.
+pub fn escaping3(item: &str, delimiter: &Delimiter) -> String {
```

Comment:
Please put the escape functions into a separate `mod`, and provide a docstring for the `mod` explaining this trick, just like in #109.  This would help future people understand the context.

Reply:
Also, `escaping3` might need a clearer name…